website/content/docs/connect/config-entries/ingress-gateway.mdx

```diff
@@ -349,13 +365,13 @@ spec:
     }
   },
   "Defaults" : {
-    "MaxConnections" : 0,
-    "MaxPendingRequests" : 0,
-    "MaxConcurrentRequests": 0,
+    "MaxConnections" : <number>,
+    "MaxPendingRequests" : <number>,
+    "MaxConcurrentRequests": <number>,
     "PassiveHealthCheck" : {
-      "interval": "<the time between checks>",
+      "interval": <the time between checks>",
```

Comment:
This reverts syntax fixes that were made in #18932.

Can you rebase this branch with the latest from 1.16.x so that you don't drop these fixes?

Reply:
Yeah I did rebase with latest 1.16.x but having trouble with the conflicts. Will redo again

Reply:
fixed